scripts/Dockerfile.pytest

```diff
@@ -0,0 +1,17 @@
+# Developmental Docker container, performing the same tasks as in .travis.yml
+FROM python:3.8
+
+WORKDIR /src
+COPY requirements.txt .
+RUN pip install -U -r requirements.txt
+
+COPY test_requirements.txt .
+RUN pip install -U -r test_requirements.txt
+
+COPY . .
+RUN pip install --no-deps -e .
+CMD pytest --cov=src/ --cov-report term-missing \
+    --cov-report term:skip-covered \
+    --cov-fail-under=100 -s -r . \
+    && flake8 src \
+    && cd docs; make html; cd ..
```

Comment:
I think if this container is just used for testing then we can forego the documentation building.

Also, is there a way to run a specific test? I'd recommend updating the README with instructions for invoking this container.

Reply:
Good suggestions @mcgalcode.  I added an ENTRYPOINT and CMD to make this easy.  I also added some docs with simple examples.